presto-main/src/main/java/com/facebook/presto/sql/planner/PlanOptimizers.java

```diff
@@ -295,6 +296,7 @@ public PlanOptimizers(
                         ImmutableSet.of(
                                 new RemoveUnreferencedScalarApplyNodes(),
                                 new TransformCorrelatedInPredicateToJoin(), // must be run after PruneUnreferencedOutputs
+                                new TransformCorrelatedNonAggregationScalarToJoin(metadata.getFunctionRegistry()),
```

Comment:
inconsistent naming: the other rule (TransformCorrelatedNoAggregationSubqueryToJoin) says `No(Aggregation`, but this one says `Non(Aggregation)`